cypress/integration/articleFlows/postSidebarActions.spec.js

```diff
@@ -19,19 +19,14 @@ describe('Post sidebar actions', () => {
 
   it('should open and close the share menu for the post', () => {
     // Check dropdown is closed by asserting the first option isn't visible
-    cy.findByRole('img', {
-      name: /^Copy article link to the clipboard$/i,
-    }).should('not.exist');
+    cy.get('#copy-post-url-button').should('not.be.visible');
```

Comment:
```suggestion
    cy.findByRole('button', { name: 'Copy Post URL' }).should('not.be.visible');
```

nitpick (blocking) - [we try to favour `findByRole` queries](https://docs.forem.com/tests/e2e-tests/), as they're less subject to change and a bit closer to how a user interacts with our app

(same with the other references below)

Reply:
@aitchiss Just did it. It seem that unlike cy,get, findByRole are incapable of finding hidden element. Took a bit too figure out

Reply:
ah yes of course I should have noticed that would be an issue in those first checks! Yeah 'findByRole' is pretty cool, because even if the element is visible to the eye, if it's not exposed in the accessibility tree (e.g. for screen reader users), then it will fail - really awesome for catching accidental accessibility bugs!

p.s. also nice work using aliases when we're doing multiple things with the button 😄